frontend/tasks/util.js

```diff
@@ -46,13 +46,13 @@ function experimentL10nId(experiment, pieces) {
   if (experiment.dev) { // For dev-only experiments, data-l10n-id=null is omitted from the DOM
     return null;
   }
-  if (typeof pieces === 'string') {
+  if (typeof pieces === "string") {
     pieces = [pieces];
   }
   const piecesOut = [];
-  for (let i=0; i<pieces.length; i++) {
+  for (let i = 0; i < pieces.length; i++) {
     piecesOut.push(pieces[i]);
-    const suffix = lookup(experiment, `${pieces.slice(0, i+1).join('.')}_l10nsuffix`);
+    const suffix = lookup(experiment, `${pieces.slice(0, i + 1).join(".")}_l10nsuffix`);
```

Comment:
Oh hey, I found a non-quote change! (It's fine though)

Reply:
Wow, sharp eye there.